contrib/windows/appveyor_build.sh

```diff
@@ -207,6 +207,6 @@ echo 'FORCE_ASSERTIONS = 1' >> Make.user
 cat Make.user
 make -j3 VERBOSE=1 all
 make -j3 VERBOSE=1 install
-make VERBOSE=1 -C examples
+make VERBOSE=1 -C test/embedding release
```

Comment:
Try this instead:

```
make VERBOSE=1 JULIA=usr/bin/julia.exe BIN=test/embedding -C test/embedding release
```

Reply:
```
make: Entering directory '/cygdrive/c/projects/julia/test/embedding'
make: usr/bin/julia.exe: Command not found
make: usr/bin/julia.exe: Command not found
cc /cygdrive/c/projects/julia/test/embedding/embedding.c -o test/embedding/embedding.exe     -lm  
make: cc: Command not found
make: *** [Makefile:36: test/embedding/embedding.exe] Error 127
make: Leaving directory '/cygdrive/c/projects/julia/test/embedding'
```

Reply:
This makefile isn't intended to be called directly from inside the Julia tree. That's why `embedding/Makefile` existed to read in the configuration variables from Make.inc and forward them to the example.

Reply:
Okay. We just need to find a way to compile the embedding test's C code while building Julia so that a compiler isn't needed to run the embedding test. (Unless that's okay.)